src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,203 @@
+{% extends "base/2019/base.html" %}
+
+{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web com os dados e as tendências do arquivo HTTP.{% endblock %}
```

Comment:
Should we leave HTTP Archive as a product name [as per the Translation guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide)

```suggestion
{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web com os dados e as tendências do HTTP Archive.{% endblock %}
```

Reply:
right, it was my fault in the last revision.